test/integration/auth/auth_test.go

```diff
@@ -23,9 +23,19 @@ package auth
 import (
 	"bytes"
 	"context"
+	"crypto/ed25519"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
 	"encoding/json"
+	"encoding/pem"
 	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
For consistency, only use `testing.T`.  None of the Kube authn/authz tests use `testify` or any other test helpers.